src/AddTodoForm.js

```diff
@@ -2,23 +2,31 @@ import React from 'react';
 // import * as reactDom from 'react-dom'
 import { useState } from "react";
 
-function AddTodoForm (props) {
- const handleAddTodo = (event) => {
-        event.preventDefault();
-     const todoTitle = event.target.title.value;
-     console.log(todoTitle, `ARE YOU HERE111`);
-     const form = document.querySelector('form');
-      form.reset();
-      props.onAddTodo(todoTitle);
 
+function AddTodoForm ({onAddTodo}) {
+
+const [todoTitle, setTodoTitle] = useState('');
+
+const handleTitleChange = (event) => {
+     const newTodoTitle = event.target.value
+     setTodoTitle(newTodoTitle)
+}
+
+ const handleAddTodo = (event) => {
+ event.preventDefault();
+ //if todoTile === '' I don't want it to return anything
+ if(todoTitle.trim().length === 0) return 
```

Comment:
I appreciate seeing a check like this to ensure that the the input value isn't an empty string or comprised of whitespace, but all that it's doing is blocking an action without informing the user why. Instead of simply returning I'd suggest adding something like an `alert` to make the UX (user experience) a bit more robust.

```suggestion
    if (todoTitle.trim().length === 0) {
      alert("Title is required");
      return;
    }

```